Formula/acmetool.rb

```diff
@@ -1,11 +1,10 @@
-require "language/go"
-
 class Acmetool < Formula
   desc "Automatic certificate acquisition tool for ACME (Let's Encrypt)"
   homepage "https://github.com/hlandau/acme"
   url "https://github.com/hlandau/acme.git",
```

Comment:
```
    homepage "https://github.com/hlandau/acmetool"
    url "https://github.com/hlandau/acmetool.git"
````

Reply:
I could if that'd make more sense here. At the time this version came out the name of the repo wasn't changed yet, so the go package name was just `github.com/hlandau/acme`. All the `-X` flags as well as the `go.mod` I added had to go with the old name. I wanted the repo name to match the package name since that's how go normally works, but if you mean that's less of a matter I'll change them.

Reply:
I personally think it makes sense to change the homepage/source urls since they redirect on GitHub with the values currently in the formula. See also https://godoc.org/github.com/hlandau/acmetool vs https://godoc.org/github.com/hlandau/acme where plain `acme` refers to `cmd/acmetool` as a legacy entrypoint.

Reply:
You are correct regarding the fact that this refactoring is the reason for the renaming of the entire package. However, for this outdated stable version this isn't the case. Users using `go build` (after having retrieved the tagged source package) would still need to use the legacy entry point with the relative path they put in their `GOPATH`. Since there isn't a variant saying `github.com/hlandau/acmetool/cmd/acmetool` in the official docs (`README.md` does not reflect these changes from v0.2.1 at all) it would be safer to retain the old repo URL if someone by any chance wants to compile this old version following the instructions in that readme, so they don't face errors with `make install` (in fact I was using `make` in an older approach of this PR) or `go build` (first knowing that `go get` wouldn't get them the wanted version).
Please refer to https://github.com/CL-Jeremy/homebrew-acmetool-v2 for my own tap containing the latest beta release which I'll submit here by June of next year, which conforms to your proposal here.

Reply:
We're not responsible for upstream documentation. If the URL changed we should change the URL in the formula.

Reply:
Okay, fair enough. I just felt a bit awkward to pass those `-X` with the old package name. Commit amended.